app/src/main/res/values/strings.xml

```diff
@@ -1,4 +1,12 @@
 <resources>
+
+    <string-array name="socialNetworks">
```

Comment:
WARNING: [UnusedResources] The resource `R.array.socialNetworks` appears to be unused
 Unused resources make applications larger and slow down builds.